python/openinference-semantic-conventions/src/openinference/semconv/trace/__init__.py

```diff
@@ -148,6 +149,46 @@ class MessageAttributes:
     """
 
 
+class MessageContentAttributes:
+    """
+    Attributes for the contents of user messages sent to an LLM
+    """
+
+    CONTENT_TYPE = "content.type"
+    """
+    The type of the content, such as "text" or "image_url".
+    """
+    CONTENT_TEXT = "content.text"
+    """
+    The text content of the message, if the type is `text`.
+    """
+    CONTENT_IMG_URL = "content.image_url"
+    """
+    The image content of the message, if the type is `image_url`.
+    An image can be made available to the model by passing a link to
+    the image or by passing the base64 encoded image directly in the
+    request.
+    """
+
+
+class ImageURLAttributes:
```

Comment:
Feels a tad openai specific. Maybe we should just create conventions around image?

Reply:
Agreed

Reply:
How about just URL and metadata for now?